numpy/core/numeric.py

```diff
@@ -2145,8 +2145,12 @@ def fromfunction(function, shape, **kwargs):
         The function is called with N parameters, where N is the rank of
         `shape`.  Each parameter represents the coordinates of the array
         varying along a specific axis.  For example, if `shape`
-        were ``(2, 2)``, then the parameters in turn be (0, 0), (0, 1),
-        (1, 0), (1, 1).
+        were ``(2, 2)``, then the parameters would in turn be
+            array([[ 0, 0 ],
+                   [ 1, 1 ]])
+        and
+            array([[ 0, 1 ],
+                   [ 0, 1 ]])
```

Comment:
Good point. It does not render quite correctly. I'll fix that. Do I close this pull request and create a new one or do I update this one somehow? I'm a novice contributor so forgive me if this should be obvious.